src/components/App/App.tsx

```diff
@@ -143,6 +147,9 @@ const App = () => {
           <Route path="/gift-a-meal-home">
             {returnComponent(Page.GiftAMeal)}
           </Route>
+          <Route path="/light-up-chinatown">
```

Comment:
I wonder if we want this to be dynamic, determined by an endpoint that lists all of the "projects"

Reply:
That's a good idea justin! Also happy to revisit this work when we get to our next "project" if it means speeding this one up

Reply:
I'm totally support making it dynamic once we have a clear design for future projects, and the backend model for "projects" :)